### PR TITLE
Create VM statuses card

### PR DESCRIPTION
**Pull request by pcbailey**
## 📝 Description

This PR adds the VM statuses card to the overview.

## Screenshots

Closed
Selection_152

Open
Selection_153

### CODE REVIEW

**Comment by pcbailey** [comment]
@avivtur @hstastna @metalice @upalatucci @vojtechszocs PTAL

File: src/views/clusteroverview/OverviewTab/vm-statuses-card/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  const errorCount = ERROR_STATUSES.reduce(
+    (acc, state) => {
+      acc.Error = acc.Error + statusCounts?.[state] || 0;
+      delete statusCounts[state];
+      return acc;
+    },
+    { Error: 0 },
+  );
+
+  statusCounts[ERROR] = errorCount.Error;
```
Comment (upalatucci) [inline src/views/clusteroverview/OverviewTab/vm-statuses-card/utils/utils.ts:64]: Can the `errorCount` be a number instead of an object?

**Review by upalatucci — COMMENTED** [review]
Great! Only this question.

File: src/views/clusteroverview/OverviewTab/vm-statuses-card/utils/utils.ts (hunk 1 above)
Comment (pcbailey) [reply src/views/clusteroverview/OverviewTab/vm-statuses-card/utils/utils.ts]: Yes and it makes it much cleaner. =) Thanks for catching that! It's been fixed.

**Review by upalatucci — APPROVED** [review]

**Pull request merged**